src/views/catalog/CreateFromInstanceTypes/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/components/UserInstanceTypeMenu/UserInstanceTypeMenu.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Dispatch, FC, SetStateAction, useMemo, useState } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { VirtualMachineClusterInstancetypeModelGroupVersionKind } from '@kubevirt-utils/models';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { ResourceIcon } from '@openshift-console/dynamic-plugin-sdk';
+import { MenuInput, MenuItem, SearchInput } from '@patternfly/react-core';
+
+type UserInstanceTypeMenuProps = {
+  items: string[];
+  selected: string;
+  setSelected: Dispatch<SetStateAction<string>>;
+};
+
+const UserInstanceTypeMenu: FC<UserInstanceTypeMenuProps> = ({ items, selected, setSelected }) => {
+  const { t } = useKubevirtTranslation();
+
+  const [searchInput, setSearchInput] = useState('');
+
+  const filteredItems = useMemo(
+    () =>
+      items.filter(
+        (opt) =>
+          isEmpty(searchInput) || opt.toLowerCase().includes(searchInput.toString().toLowerCase()),
+      ),
+    [items, searchInput],
+  );
+
```

Comment:
i would add here 

if isEmpty(filteredItems) return (
        <MenuItem isDisabled>{t('No results found')}</MenuItem>
      )

Reply:
with the menu component